### ISSUE TITLE
Improvement: add support for not fixed item heights

### ISSUE THREAD

**Issue by marcin-michalek** [Azoft/CarouselLayoutManager#31]
Hello,

thanks for a great library! Do you have any thoughts on how to use it on not fixed item height? For instance I have an item view with multiple text views which are generated in the adapter based on adapter data. If item view is higher than the defined fixed height the item view is cut off at the bottom.

**Comment by mig35** [comment]
@marcin-michalek we don't think we will add this support soon

**Comment by Kisty** [comment]
I think the [`RecyclerView#setHasFixedHeight()`](https://github.com/Azoft/CarouselLayoutManager/blob/master/CarouselLayoutManager/sample/src/main/java/com/azoft/carousellayoutmanager/sample/CarouselPreviewActivity.java#L80) call is actually applying to the `RecyclerView`, not the item height. Perhaps the item height can be changed but not the `RecyclerView`?

**Comment by Kisty** [comment]
The way I'd do it is to get all the data upfront and manually measure text height before setting the `Adapter` to the `RecyclerView`.

**Comment by marcin-michalek** [comment]
Yep - that's how I did it. Foretunatelly, I had a max of three items in the carousel.
